modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,228 @@
++
+. `KEY_VALUE`: Creates an Iceberg table with a `Key` column and a `Value` column. Redpanda stores the raw topic content in the `Value` column.
+. `VALUE_SCHEMA_ID_PREFIX`: Creates an Iceberg table whose structure matches the Redpanda schema for this topic, with columns corresponding to each field. Redpanda parses the topic values per field and stores them in the corresponding table columns. 
+. `DISABLED` (default): Disables the creation of an Iceberg table for this topic.
++
+[,bash]
+----
+rpk topic alter-config <new-topic-name> --set redpanda.iceberg.mode=<topic-iceberg-mode>
+----
++
+[,bash,role=no-copy]
+----
+TOPIC            STATUS
+new-topic-name   OK
+----
+
+. Register a schema for the topic (optional). You must also set the `redpanda.iceberg.mode` topic property to `VALUE_SCHEMA_ID_PREFIX`. Redpanda uses xref:manage:schema-reg/schema-id-validation.adoc#set-subject-name-strategy-per-topic[`TopicNameStrategy`] (`<topic-name>-value` or `<topic-name>-key`) to find the matching schema based on the subject name. Redpanda derives the Iceberg table format from the schema and writes data files to object storage in this format. Protobuf and Avro schemas are supported.
++
+[,bash]
+----
+rpk registry schema create <subject-name> --schema </path-to-schema> --type <format>
+----
++
+[,bash,role=no-copy]
+----
+SUBJECT                VERSION   ID   TYPE
+new-topic-name-value   1         1    PROTOBUF
+----
++
+If you select `KEY_VALUE` for the topic Iceberg mode, Redpanda uses a simple schema consisting of the record’s key, value, and the original record timestamp. Consider this schemaless approach if you can use the data in a more semi-structured format.
++
+NOTE: Redpanda automatically adds three Kafka-related fields to the table it creates: the record timestamp, the partition it belongs to, and its offset.
+
+As you produce records to the topic, the data also becomes available in object storage for consumption by Iceberg-compatible clients. 
+
+== Consume data in Iceberg tables
+
+You can use the same analytical tools to access table data in a data lake as you would for a relational database.
+
+For example, suppose you produce the same stream of events to a topic `test_topic`, and another topic `test_topic_schemaless`. The topics have Tiered Storage configured to an AWS S3 bucket. A sample record contains the following data:
+
+[,bash,role=no-copy]
+----
+{'Topic': 'apple', 'Sentiment': 'positive', 'TweetId': '126377656416612353', 'TweetDate': 'Tue Oct 18 19:22:35 +0000 2011', 'TweetText': 'Great up close & personal event @Apple tonight in Regent St store!'}
+----
+
+When you point your Iceberg-compatible tool or framework to the object storage location of the Iceberg tables, how you consume the data depends on whether you've registered a schema on which to base the table format. In either approach, you do not need to rely on complex ETL jobs or pipelines to consume real-time data from Redpanda.
+
+=== Structured approach with schema
```

Comment:
Obviously some of these examples will be different now with the new schema/column handling and will need updating @kbatuigas @rockwotj

Reply:
Yes happy to work with you @kbatuigas on this